src/sparseml/exporters/transforms/kv_cache/configs.py

```diff
@@ -102,20 +105,24 @@ class Config:
     multiply_batch_by_num_att_heads=False,
 )
 
-BLOOM_CONFIG = KeyValueCacheConfig(
-    model_name="bloom",
-    additional_transforms=None,
-    key_num_attention_heads="num_attention_heads",
-    key_num_embedding_hidden_size="n_embed",
+MPT_CONFIG = KeyValueCacheConfig(
+    model_name="mpt",
+    additional_transforms=AdditionalTransformsMPT,
+    key_num_attention_heads="n_heads",
+    key_num_embedding_hidden_size="d_model",
     transpose_value_input=None,
     transpose_key_input=(0, 1, 3, 2),
-    multiply_batch_by_num_att_heads=True,
+    multiply_batch_by_num_att_heads=False,
 )
 
 
 def get_kv_cache_config(
     model_path: str,
-    supported_configs: List[BaseModel] = [OPT_CONFIG, CODEGEN_CONFIG, BLOOM_CONFIG],
```

Comment:
We need to remove BLOOM, we are not supporting it officially in the nightly.

Reply:
if we have some internal testing, hidden support seems ok